src/docs/patterns/overlay.mdx

```diff
@@ -1,73 +1,132 @@
-import { DemoCard, ImageGrid, MaterialDesignDescription } from '../../app/components';
-import BasicBottomsheet from '../../app/assets/design-patterns/action-list/action-list-overlay.png';
-import ComplexBottomsheet from '../../app/assets/design-patterns/bottom-sheet/complex-bottomsheet.png';
+import { DemoCard, ImageGrid, MaterialDesignDescription, Divider } from '../../app/components';
+import Banner from '../../app/assets/design-patterns/overlays/overlay-banner.svg';
+import DesktopDialog from '../../app/assets/design-patterns/overlays/desktop-dialog.png';
+import DesktopDialogTask from '../../app/assets/design-patterns/overlays/desktop-dialog-task.png';
+import SimpleBottomsheet from '../../app/assets/design-patterns/overlays/simple-bottomsheet.png';
 
-<!-- KEYWORDS: design pattern overlay modal sidebar popup dialog -->
+<!-- KEYWORDS: design pattern overlay modal sidebar popup dialog side sheet -->
 
-# Overlay Patterns
+# Overlays
 
-Overlays are used to present an immediate interaction to a user. These can be used for presenting information (such as with a Dialog) or capturing it (such as in a form). These elements come in a variety of shapes and sizes and can be either blocking (the user must interact with the overlay before they can do anything else) or non-blocking.
+<ImageGrid images={[Banner]} regularWidth />
 
-## Modal Dialog
+Overlays are useful in a variety of situations when you need to provide more context or information to a user than can be found in your primary content. For example, they can be used to present a confirmation dialog, details about a selected item, or actions that can be taken on a page. These can be used for presenting information (such as with a dialog) or capturing it (such as in a form).
+
```

Comment:
The opening paragraph shall be more eye-catchy and summarize the whole paragraph. This whole content belongs to a when-to-use / Common Use Case section

Reply:
For the first sentence, I had to read twice to understand what's going on. Consider breaking it down further.